scripts/db.sh

```diff
@@ -2,7 +2,7 @@
 
 print_usage_and_exit() {
   echo "Usage: $0 [dump|import|drop_and_import] filename [args_for_mysql]"
-  echo "For example: $0 --user=USER --password=PASS --host=HOST"
+  echo "For example: $0 import mysql-dump-2015-03-15.sql --user=USER --password=PASS --host=HOST"
```

Comment:
This isn't a great filename, as this script is meant to deal with `*.tar.gz` files.